Versioning.md

```diff
@@ -0,0 +1,76 @@
+# Versioning
+
+This document describes the versioning policy for this repository.
+
+## Goals
+
+### API Stability
+
+Once the API for a given signal (spans, logs, metrics, baggage) has been officially released, that API module will function with any SDK that has the same major version, and equal or greater minor or patch version.
```

Comment:
`with any SDK` - if we add new API in v1.2, does it have to be supported by v1.1 .. or do we only guarantee forward-compatibility for a major version? If we don't, does it mean that for newly added classes (which does not break older apps that would use only the old classes) we need to respin and publish the new major version?

Reply:
Not sure if I understood the question, but if there is any `minor` version bump (v1.1 to v1.2) for api due to adding new class interface, and any library is instrumented/compiled with v 1.2, it should work fine with any sdk ( i.e, shared library, dll ) with same major version  ( 1 here ), and equal or greater minor version ( 2 here). 

Sorry I didn't answer your question - would need you to elaborate  : )

Reply:
Maybe remove comma before `and` - you usually put a `comma` before `and` when it’s connecting two independent clauses. In this case I think it would be nicer to elaborate or use something like **v1.x** and **v1.y** , where minor **y>=x** . 
```suggestion
Once the API for a given signal (spans, logs, metrics, baggage) has been officially released, that API module will function with any SDK that has the same MAJOR version and equal or greater MINOR or PATCH version. For example, application compiled with API v1.1 is compatible with SDK v1.1.2, v1.2.0, etc.
```